.github/workflows/docker-deploy.yml

```
@@ -0,0 +1,125 @@
+        uses: docker/setup-buildx-action@v3
+
+      - name: Build Docker Image
+        run: |
+          DOCKER_BUILDKIT=1 \
+          VITE_AUTO_LOGIN=false \
+          VITE_CLERK_AUTH_ENABLED=true \
+          VITE_CLERK_PUBLISHABLE_KEY=${{ secrets.CLERK_PUBLISHABLE_KEY }} \
+          make docker_build
+
+      - name: Cleanup all Docker Hub tags
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:38:20: Double quote to prevent globbing and word splitting

(shellcheck)